packages/playwright-ct-core/package.json

```
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "playwright-core": "1.51.0-next",
-    "vite": "^5.4.14",
+    "vite": "^6.0.0",
```

```suggestion
    "vite": "^5.4.14 || ^6.0.0",
```

maybe like that for now?